mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
-  return Math.random() * 10;
+  return Math.random() * 10;      
 }
+// The function above will return a random integer from 0 and 9, The random() method returns a random number from 0 (inclusive) up to but not including 1 (exclusive)
 
```

Comment:
Hi Hozan, when if you run `Math.random` what output do you get? If you multiple the result of `Math.random` with 10 does it result in an integer?

Reply:
Hi @louisechow , many thanks for the feedback.
You are right, I should have worded my answer properly.
The `Math.random` method will return a floating point number between 0 and 1 (not included), and when it is multiplied by 10 we will still get a floating point number between 0 and 10 (not included).